cpp/dolfinx/io/ADIOS2Writers.h

```diff
@@ -960,6 +975,27 @@ class VTXWriter : public ADIOS2Writer
     impl_adios2::define_attribute<std::string>(*_io, "vtk.xml", vtk_scheme);
   }
 
+  /// @brief Create a VTX writer for a list of fem::Functions using
+  /// the default ADIOS2 engine.
+  ///
+  /// This format supports arbitrary degree meshes.
+  ///
+  /// @param[in] comm The MPI communicator to open the file on
+  /// @param[in] filename Name of output file
+  /// @param[in] u List of functions. The functions must (1) share the
+  /// same mesh and (2) be (discontinuous) Lagrange functions. The
+  /// element family and degree must be the same for all functions.
+  /// @param[in] mesh_policy Controls if the mesh is written to file at
+  /// the first time step only or is re-written (updated) at each time
+  /// step.
+  /// @note This format supports arbitrary degree meshes.
+  VTXWriter(MPI_Comm comm, const std::filesystem::path& filename,
+            const typename adios2_writer::U<T>& u,
+            VTXMeshPolicy mesh_policy = VTXMeshPolicy::update)
+      : VTXWriter(comm, filename, u, "BPFile", mesh_policy)
+  {
+  }
```

Comment:
I think we can remove this, as it could rather be a default argument of the previous constructor.

Reply:
As I mentioned, in the Fides PR I used the approach that you recommend and it was heavily opposed on the grounds that

> a constructor shall not have more than one default argument.

I eventually changed it back then and now I just reused the same "idiom".